clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/servergroup/CloneOpenstackAtomicOperation.groovy

```diff
@@ -96,6 +96,7 @@ class CloneOpenstackAtomicOperation implements AtomicOperation<DeploymentResult>
       scaleup = description.serverGroupParameters?.scaleup ?: ancestorParams.scaleup
       scaledown = description.serverGroupParameters?.scaledown ?: ancestorParams.scaledown
       tags = description.serverGroupParameters?.tags ?: ancestorParams.tags
+      asgResourceFilename = description.serverGroupParameters?.asgResourceFilename ?: ancestorParams.asgResourceFilename
```

Comment:
This shouldn't go in the server group parameters object - a user will never override it - it will only change if your load balancer choice differs from the group you are cloning.